ui/raidboss/data/03-hw/raid/a8s.js

```diff
@@ -519,8 +538,9 @@
       condition: Conditions.targetIsYou(),
       durationSeconds: 8,
       alertText: {
-        en: 'Min HP: Provoke Boss => Late NE Tornado',
+        en: 'Min HP: Provoke Boss => Late NW Tornado',
```

Comment:
I understand, it can be NE or NW. Should we change the indications?

-  Cyclon NE
![Cyclon NE](https://user-images.githubusercontent.com/61565834/82569432-894f0d80-9b80-11ea-98a2-18bced6b83cd.PNG)

- Cyclon NW
![Cyclon West](https://user-images.githubusercontent.com/61565834/82569473-95d36600-9b80-11ea-9863-ff36270ae350.PNG)

Reply:
Yes, either one is fine.  However, I don't understand why you were changing it.  The existing choice seemed good enough, and matches the guide linked in the timeline file.

Reply:
I restore